common/model-views.h

```diff
@@ -57,6 +69,55 @@ inline ImVec4 blend(const ImVec4& c, float a)
 
 namespace rs2
 {
+    struct textual_icon
+    {
+        explicit constexpr textual_icon(const char (&unicode_icon)[4]) :
```

Comment:
Is the compiler going to enforce this and reject something of more length? 
Why not just use an `std::string` with implicit conversion to `char*`?

Reply:
Yes, the compiler will not accept strings with more or less than 4 chars.
You mean why not take a string in the constructor? and hold it as member?